app/views/patients/_change_log.html.erb

```diff
@@ -11,7 +11,7 @@
     </thead>
 
     <tbody>
-      <% patient.versions.each do |revision| %>
+      <% patient.all_versions(current_user.allowed_data_access?).each do |revision| %>
```

Comment:
why only admins & data users getting fulfillment info?

Reply:
we generally only show fulfillment data (in the UI via the partial, for example) if someone's a data vol or above - if you aren't a data vol or above you can't do stuff like see the accounting pane, the fulfillment partial, or run a data export. This gates it according to our existing standard.